packages/buildbucket-dart/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
-name: buildbucket_proto
+name: buildbucket_pb
```

Comment:
If we plan to support creating Dart clients for this, should we just call it buildbucket?

Reply:
Changed to buildbucket